packages/wp-now/src/wp-now.ts

```diff
@@ -233,23 +248,36 @@ export default class WPNow {
 			this.php.mount(this.options.projectPath, this.options.documentRoot);
 			return;
 		}
-		// Mode: core, plugin or theme
+
+		// Mount wordpress in all modes except index
 		this.mountWordpress();
 		const { wpContentPath } = this.options;
 		fs.ensureDirSync(wpContentPath);
-		fs.copySync(
-			path.join(
-				WORDPRESS_VERSIONS_PATH,
-				'latest',
-				'wordpress',
-				'wp-content'
-			),
-			wpContentPath
-		);
-		this.php.mount(
-			wpContentPath,
-			`${this.options.documentRoot}/wp-content`
-		);
+
+		// Mode: wp-content - mount the wp-content folder as is
+		if (mode === 'wp-content') {
```

Comment:
At this point it makes sense to turn all these magic strings into consts like `WpNow.MODE_WP_CONTENT`, otherwise it's hard to remember and easy to make typos. Not a blocker for this PR, but it would be a good follow-up.

Reply:
Well, I guess having a WPNowMode type makes it better. Feel free to make a call on this one.

Reply:
Why not both? I've changed the string literal union to the enum. Thanks for idea!